cmd/http-check/http-check.yaml

```diff
@@ -9,17 +8,23 @@ spec:
   timeout: 10m
   podSpec:
     containers:
-    - name: http
-      image: kuberhealthy/http-check:v1.2.2
-      imagePullPolicy: IfNotPresent
-      env:
-        - name: CHECK_URL
-          value: "http://google.com"
-      resources:
-        requests:
-          cpu: 15m
-          memory: 15Mi
-        limits:
-          cpu: 25m
-      restartPolicy: Always
+      - name: http
+        image: kuberhealthy/http-check:v1.2.4
+        imagePullPolicy: IfNotPresent
+        env:
+          - name: CHECK_URL
+            value: "http://google.com"
+          - name: COUNT
+            value: "10"
+          - name: SECONDS
+            value: "1"
+          - name: PASSING
```

Comment:
Can we switch this to `PASSING_PERCENT` and change the value to a whole number?

Reply:
I have changed this to `PASSING_PERCENT`